nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -216,6 +217,12 @@ in {
         default = true;
       };
 
+      changeMaxTootChars = lib.mkOption {
+        description = "Change maximum allowed character count. Default 500 character count.";
+        type = lib.types.nullOr lib.types.int;
+        default = null;
```

Comment:
This is a bit confusing why the default is null but the text tells you 500.

Reply:
The `MAX_TOOT_CHARS` parameter is responsible for the number of characters. If it is not in the variables, then the `MAX_CHARS` value will be by default - 500 characters.

Reply:
Don't explain this to me but instead in the option.

Reply:
This variant?
```
        type = lib.types.int;
        default = 500;
```